assets/charts/components/prometheus-operator/CONTRIBUTING.md

```diff
@@ -1,10 +1,12 @@
 # Contributing Guidelines
```

Comment:
In this first commit, can you provide the version of chart and app for each sub chart? Also the commit says update prometheus-operator but we don't update that.